grafonnet-base/util/grid.libsonnet

```diff
@@ -131,4 +131,96 @@ local d = import 'github.com/jsonnet-libs/docsonnet/doc-util/main.libsonnet';
         panels: root.makePanelGrid(panelsBeforeRowGroups, panelWidth, panelHeight, 0),
       }
     ).panels,
+
+  '#wrapPanels':: d.func.new(
+    |||
+      `wrapPanels` returns an array of `panels` organized in a grid, wrapping up to next 'row' if total width exceeds full grid of 24 columns.
+      'panelHeight' and 'panelWidth' are used unless panels already have height and width defined.
+    |||,
+    args=[
+      d.arg('panels', d.T.array),
+      d.arg('panelWidth', d.T.number),
+      d.arg('panelHeight', d.T.number),
+      d.arg('startY', d.T.number),
+    ],
+  ),
+  wrapPanels(panels, panelWidth=8, panelHeight=8, startY=0):
+    std.foldl(
+      function(acc, panel)
+        // handle row
+        if panel.type == 'row'
+        then
+          local shiftY = acc.cursor.maxH;
```

Comment:
Perhaps don't substitute `maxH` with `shiftY`, it took me bit to understand what maxH was used for until I noticed this local.

Reply:
Dropped shiftY